service/src/main/java/greencity/rating/RatingCalculation.java

```diff
@@ -30,10 +25,9 @@ public class RatingCalculation {
      * @param userVo of {@link UserVO}
      */
     public void ratingCalculation(RatingCalculationEnum rating, UserVO userVo) {
-        String accessToken = httpServletRequest.getHeader(AUTHORIZATION);
         User user = modelMapper.map(userVo, User.class);
         userVo.setRating(userVo.getRating() + rating.getRatingPoints());
-        restClient.save(userVo, accessToken);
```

Comment:
I see we use` restClient.save(userVO)` in `AchievementServiceImpl save`
Should it also be replaced by `userService.save(userVo)`?

Reply:
replaced